README.md

```diff
@@ -1 +1,30 @@
 # go-indexer-core
+[![Go Reference](https://pkg.go.dev/badge/github.com/filecoin-project/go-indexer-core.svg)](https://pkg.go.dev/github.com/filecoin-project/go-indexer-core)
```

Comment:
are any of the other badges from https://github.com/ipld/go-car/blob/master/README.md worth copying?

Reply:
Probably, added some.